openmdao/utils/mpi.py

```diff
@@ -123,6 +127,38 @@ def __init__(self):
         self.size = 1
 
 
+@contextmanager
+def MultiProcFailCheck(comm):
```

Comment:
minor detail, but should this be snake case since it is a function?

Reply:
Fixed